arangod/Aql/Query.cpp

```diff
@@ -1482,11 +1482,10 @@ void Query::enterState(QueryExecutionState::ValueType state) {
 
 /// @brief cleanup plan and engine for current query
 ExecutionState Query::cleanupPlanAndEngine(ErrorCode errorCode, bool sync) {
-  ensureExecutionTime();
```

Comment:
It seems that removing the unconditional update of the execution time causes assertion failures in the tests.